blazor/pdfviewer2/getting-started/server-side-application.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Getting Started with PDF Viewer in Blazor Server App | Syncfusion
+description: Learn how to getting started with PDF Viewer control in Blazor Server-side application. You can view and comment on PDFs in ease and also can fill fields. 
+platform: Blazor
+control: PDF Viewer2
+documentation: ug
+---
+
+# Getting Started with Blazor PDF Viewer Component in Blazor Server App
+
+This section briefly explains about how to integrate [Blazor PDF Viewer](https://www.syncfusion.com/blazor-components/blazor-pdf-viewer) component in your Blazor Server App using Visual Studio.
+
+## Prerequisites
+
+* [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
+
+## Integrate PDF Viewer into Blazor Server App
+
+1. Start Visual Studio and select **Create a new project**.
+
+2. For a Blazor Server experience, choose the **Blazor Server App** template. Select **Next**. 
+![Create-new-blazor-server-app](GettingStarted_images/start-window-create-new-project.png)
+
+3. Provide a **Project Name** and confirm that the *Location* is correct. Select Next. 
+![Set-project-name](GettingStarted_images/Set-project-name.png)
+
+4. In the **Additional information** dialog, set the target framework. 
+![Set-target-framework](GettingStarted_images/Additional_information.png)
+
+## Install Blazor PDF Viewer NuGet package in Blazor Server App
+
+To add Blazor PDF Viewer component in Blazor Server App, use `SfPdfViewer` component and theme style sheet in corresponding NuGet based on the operating system of the server you intend to host, as shown below.
+
+* For **Windows**, use [Syncfusion.Blazor.SfPdfViewer.Windows](https://www.nuget.org/packages/Syncfusion.Blazor.SfPdfViewer.Windows) and [Syncfusion.Blazor.Themes](https://www.nuget.org/packages/Syncfusion.Blazor.Themes/)

```

Comment:
We have only one package. We don't have individual packages for different environment.